### PR TITLE
fix: send script UID to server

### PR DESCRIPTION
**Pull request by Seavenly**

### CODE REVIEW

File: src/old/messages/components/Modal/component.js [hunk 1]
```diff
@@ -125,6 +126,11 @@ export default getGlobalComponent('__paypal_credit_modal_old__', () =>
                 queryParam: true,
                 sendToChild: false,
                 value: getTargetMeta
+            },
+            scriptUID: {
```
Comment (gregjopa) [inline src/old/messages/components/Modal/component.js:130]: This usage of `getCurrentScriptUID` looks good to me. We can see that zoid uses the same function when defining the namespace: https://github.com/krakenjs/zoid/blob/6ae489c32c6f605230be070801a92ddf62f0a1dd/src/lib/global.js#L8

**Review by gregjopa — APPROVED** [review]
Looks good to me!

**Review by gillycheesesteak — APPROVED** [review]

**Review by bluepnume — APPROVED** [review]
LGTM!

**Pull request merged**